### PR TITLE
Avoid races in floating IPs test.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: deploy/shakenfist_ci/tests/test_floating_ips.py [hunk 1]
```diff
@@ -63,6 +63,17 @@ def test_simple(self):
...
+        attempts = 0
+        for _ in range(6):
+            try:
+                r = requests.request(
+                    'GET', 'http://%s/' % ifaces[0]['floating'])
+                self.assertEqual(200, r.status_code)
+                self.assertEqual('Floating IPs work!\n', r.text)
+
+            except Exception as e:
+                if attempts < 5:
```
Comment (mandoonandy) [inline deploy/shakenfist_ci/tests/test_floating_ips.py:76]: probably should increment `attempts`

Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_floating_ips.py:76]: Y U so hater? Also yes.

Comment (mandoonandy) [reply deploy/shakenfist_ci/tests/test_floating_ips.py:77]: Sorry to be a pain, but why it does it test 6 times regardless of the outcome?

Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_floating_ips.py:77]: Because I am bad at computers.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**